conference/NW/ipsn.yaml

```diff
@@ -1,15 +0,0 @@
-- title: IPSN
-  description: ACM/IEEE International Conference on Information Processing in Sensor Networks
-  sub: NW
-  rank: B
-  dblp: ipsn
-  confs:
-    - year: 2022
-      id: ipsn22
-      link: https://ipsn.acm.org/2022/
-      timeline:
-        - abstract_deadline: '2021-10-22 23:59:59'
-          deadline: '2021-10-29 23:59:59'
-      timezone: UTC-12
-      date: May 4-6, 2022
-      place: Milan, Italy
```

Comment:
Why this file was deleted?

Reply:
it is a duplicated file. there is a file named ipsn.yml already in the folder